### ISSUE TITLE
Following threads

### ISSUE THREAD

**Issue by tio-trom** [friendica/friendica#11687]
After you introduced the latest options for notifications, users can select to only get notifications when someone replies to their own comments/posts. But now I wanted to also be able to follow some threads and get all of the activity there. However it seems I cannot since that means I can only see the activity in threads filtered through my notification settings. And so, if I chose to only notify me when someone replies on my posts/comments, then following a thread will only send you notifications when someone replies to your comments on that thread....

I think there should be an option to follow a thread fully, regardless of the notification settings.

**Comment by annando** [comment]
Have you tested if this currently doesn't work? Because at time of coding, I especially had these following of threads in mind. Means: When you follow a thread or when you marked for a contact to get notified about new posts, this should override all notifications settings for that case.

**Comment by tio-trom** [comment]
Interesting. I will test more but what I did was to "like" a thread. Then I realized it also marked it as "following the thread" so I could not even follow. And of course I was not notified of any comments. Maybe this is a tiny bug. I will report back - I have now freshly followed other threads.

**Comment by annando** [comment]
Ah yeah, I understand the problem. The system believes that you don't need to additionally follow the thread when you already interacted. Of course with the new settings you should always be able to follow a thread.

**Comment by tio-trom** [comment]
Ok so if you like a post you can't "properly" follow it then?

**Comment by annando** [comment]
You said so, I haven't checked that.

**Comment by tio-trom** [comment]
> You said so, I haven't checked that.

Yes that I tested for sure. If I click "like" on a post, it automatically selects it as "followed" with the notification settings I have for my account.

**Comment by annando** [comment]
To avoid any misunderstanding: Technically we can initiate a "follow" at any point in time. So when the system currently says that you are already following when you simply interacted, then we should change this. This is just an optical glitch.

**Comment by tio-trom** [comment]
Ah I see. Yes this needs to be changed so that you can interact but also follow. So that the interaction won't overwrite the follow action.

**Comment by tokenwizard** [comment]
I just stumbled on this discussion after posting #12346 which may be related.

Is there a way to adjust this behavior in the Admin panel? I would like for my instance to not automatically follow a post/thread just because I Like it.
